spec/lib/percy/capybara/client_spec.rb

```diff
@@ -4,45 +4,59 @@
     capybara_client = Percy::Capybara::Client.new(client: client)
     expect(capybara_client.client).to eq(client)
   end
+
   describe '#enabled?' do
-    before(:each) do
-      @original_env = ENV['TRAVIS_BUILD_ID']
-      ENV['TRAVIS_BUILD_ID'] = nil
-    end
-    after(:each) do
-      ENV['TRAVIS_BUILD_ID'] = @original_env
-      ENV['PERCY_ENABLE'] = nil
-    end
+    context 'when required environment variables set' do
+      before(:context) do
+        set_required_env_variables
+      end
 
-    context 'in supported CI environment' do
-      it 'is true' do
-        ENV['TRAVIS_BUILD_ID'] = '123'
+      after(:context) do
+        clear_required_env_variables
+      end
+
+      it 'is true when PERCY_ENABLE is 1' do
+        ENV['PERCY_ENABLE'] = '1'
         expect(Percy::Capybara::Client.new.enabled?).to eq(true)
       end
+
+      it 'is true when PERCY_ENABLE is not set' do
+        ENV.delete 'PERCY_ENABLE'
+        expect(Percy::Capybara::Client.new.enabled?).to eq(true)
+      end
+
+      it 'is false when PERCY_ENABLE is 0' do
+        ENV['PERCY_ENABLE'] = '0'
+        expect(Percy::Capybara::Client.new.enabled?).to eq(false)
+      end
     end
-    it 'is true by default for local dev environments and unknown CI environments' do
-      expect(Percy::Capybara::Client.new.enabled?).to eq(true)
-    end
-    it 'is true if PERCY_ENABLE=1 is set' do
-      ENV['PERCY_ENABLE'] = '1'
-      expect(Percy::Capybara::Client.new.enabled?).to eq(true)
-    end
-    it 'is true if PERCY_ENABLE isn\'t set' do
-      ENV.delete('PERCY_ENABLE')
-      expect(Percy::Capybara::Client.new.enabled?).to eq(true)
-    end
-    it 'is false if PERCY_ENABLE=0 is set' do
-      ENV['PERCY_ENABLE'] = '0'
-      expect(Percy::Capybara::Client.new.enabled?).to eq(false)
+
+    context 'when required environment variables not set' do
+      before(:each) do
+        clear_required_env_variables
+      end
+
+      it 'is false' do
+        ENV.delete 'PERCY_ENABLE'
+        expect(Percy::Capybara::Client.new.enabled?).to eq(false)
+      end
+
+      it 'is false when PERCY_ENABLE is 1' do
+        ENV['PERCY_ENABLE'] = '1'
+        expect(Percy::Capybara::Client.new.enabled?).to eq(false)
+      end
     end
   end
+
```

Comment:
I don't personally find that _more_ readable, but I don't mind much either way on this one (standard personal preference thing). 🐰  Since this PR is not about changing the style of this gem, can you revert the newline changes to keep things internally consistent and just to the change at hand? We can create a long-term ruby styleguide / install a linter later.